android/pom.xml

```diff
@@ -6,7 +6,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.google.guava</groupId>
   <artifactId>guava-parent</artifactId>
-  <version>HEAD-android-SNAPSHOT</version>
+  <version>1.0-HEAD-android-SNAPSHOT</version>
```

Comment:
Versions are bumped to this style to fix version number conflicts with the module system. This value needs to start with `N.N-`, where `N` is an integer. Previously discussed and cleared as non-risk